binarytree/__init__.py

```diff
@@ -73,6 +96,126 @@ def _add_left(parent, child):
 def _add_right(parent, child):
     """Add the child to the right of the parent."""
     setattr(parent, _right_attr, child)
+    
+def pruneLeft(node):
+    """Prunes the left subtree of a node."""
+    node.__setattr__(_left_attr, _null)
+        
+def pruneRight(node):
+    """Prunes the right subtree of a node."""
+    node.__setattr__(_right_attr, _null)
+
+def getParentNode(root,nodeId):
+    """Recovers the parent node from a given node, if exists"""    
+    current_nodes = [root]
+    level_not_empty = True
+    while level_not_empty:
+        level_not_empty = False
+        next_nodes = []
+        for node in current_nodes:
+            if node.left != _null:
+                level_not_empty = True
+                next_nodes.append(node.left)
+                if node.left.id == nodeId:
+                    return node
+            if node.right != _null:
+                level_not_empty = True                
+                next_nodes.append(node.right) 
+                if node.right.id == nodeId:
+                    return node            
+        current_nodes = next_nodes                
+    return
+    
+def getNodeById(root,nodeId):
```

Comment:
Please change from camelcase to snakecase for consistency. Missing a whitespace after comma.